--- types/argument.py 2021-05-01 21:40:03.454709343 +0000
+++ types/argument.py  2021-05-01 21:40:10.890839788 +0000
@@ -76,6 +76,8 @@
     from .field import Field
     from .inputfield import InputField
     if extra_args:
+        if 'headers' in extra_args:
+            del extra_args['headers']
         extra_args = sorted(extra_args.items(), key=lambda f: f[1])
     else:
         extra_args = []
